### DISCUSSION TITLE
Problem listing with Internet-Radio yp. Multiple radio accounts on one server

### DISCUSSION THREAD

**Discussion by cooper-fm** [AzuraCast/AzuraCast#7448] [General]
Hi,

I'm currently building a small range of stations for different genres. Whilst we have no problems listing multiple stations over at Xiph using YP, unfortunately each time we add a new station the last entry for the Internet-Radio YP directory is replaced with the current one.

For example, there's around 8 stations listed on Xiph, only one listed on Internet-Radio.

We are transitioning from Centova on a bare metal server (where all of this worked without any problems), to AzuraCast. Other than that, we can't identify any other changes that might cause an issue.

If anyone could help us look in the right direction, that would be great,

Thanks

**NOTES**
We have raised this query with Internet-Radio - you can see the comments here: https://www.internet-radio.com/community/threads/when-adding-a-new-station-the-earlier-station-is-removed-from-internet-radio-yp.53377/

**Comment by Vaalyn** [comment]
From the linked thread it sounds like a WebHook would be the best solution for this and should be pretty straight forward to implement if we can get all the needed info regarding what needs to be sent where and in what format for them to be able to work with it.
